internal/tool/multi/deployer.go

```diff
@@ -405,16 +394,13 @@ func (d *deployer) registerReplica(g *group, info *protos.WeaveletInfo) error {
 
 // HandleLogEntry implements the envelope.EnvelopeHandler interface.
 func (d *deployer) HandleLogEntry(_ context.Context, entry *protos.LogEntry) error {
-	d.logSaver(entry)
+	d.logsDB.Add(entry)
```

Comment:
wouldn't this panic if logsDB is nil?

Somehow it assumes that someone created a deployer with newDeployer.

Reply:
That's true! But, I don't think any of a deployer's methods would work if you don't construct it with `newDeployer`, so I think it's okay?